hazelcast/src/main/java/com/hazelcast/internal/diagnostics/MultiLineDiagnosticsLogWriter.java

```diff
@@ -16,14 +16,16 @@
 
 package com.hazelcast.internal.diagnostics;
 
+import java.util.Locale;
+
 import static com.hazelcast.util.StringUtil.LINE_SEPARATOR;
 
 /**
  * A {@link DiagnosticsLogWriter} that writes over multiple lines. Useful for human reading.
  */
 class MultiLineDiagnosticsLogWriter extends DiagnosticsLogWriter {
 
-    private static final String STR_LONG_MIN_VALUE = String.format("%,d", Long.MIN_VALUE);
+    private static final String STR_LONG_MIN_VALUE = String.format(Locale.US, "%,d", Long.MIN_VALUE);
```

Comment:
This is also fine but as a minor comment, we have [this one] (https://github.com/hazelcast/hazelcast/blob/master/hazelcast/src/main/java/com/hazelcast/util/StringUtil.java#L40) to get locale for internal usage. It may be better to get internal-locale from one place.